MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/ReflCommandAdapter.h

```diff
@@ -52,31 +55,25 @@ class ReflCommandAdapter : public QObject {
       auto &child = m_adaptee->getChild();
       for (auto &ch : child) {
         m_adapter.push_back(Mantid::Kernel::make_unique<ReflCommandAdapter>(
-            submenu, std::shared_ptr<ReflCommand>(ch.get())));
+            submenu, std::move(ch)));
```

Comment:
Yes. You've definitely fixed a resource leak here. It might be worth you speaking to me about this if you don't understand how it happened.